frontend/javascripts/oxalis/api/api_latest.ts

```diff
@@ -673,16 +673,24 @@ class TracingApi {
     }
 
     const segmentationLayerName = api.data.getSegmentationLayerNames()[0];
+    const layer = getLayerByName(Store.getState().dataset, segmentationLayerName);
+
+    const resolutionInfo = getResolutionInfo(layer.resolutions);
+    const finestResolution = resolutionInfo.getFinestResolution();
+    // By default, getDataForBoundingBox uses the finest existing magnification.
+    // We use that as strides to traverse the data array properly.
+    const [dx, dy, dz] = finestResolution;
+
     const data = await api.data.getDataForBoundingBox(segmentationLayerName, {
       min,
       max,
     });
```

Comment:
Considering your fix means `api.data.getDataForBoundingBox` returns the segments in the current mag?